README.md

```diff
@@ -120,13 +122,37 @@ InvisibleCaptcha.setup do |config|
   # config.timestamp_threshold = 4
   # config.timestamp_enabled   = true
   # config.injectable_styles   = false
+  # config.ip_enabled          = true
+  # config.secret              = REQUIRED_SECRET_KEY
 
   # Leave these unset if you want to use I18n (see below)
   # config.sentence_for_humans     = 'If you are a human, ignore this field'
   # config.timestamp_error_message = 'Sorry, that was too quick! Please resubmit.'
 end
 ```
 
+#### IP Enabled Setup
+
+View code:
+
+```erb
+<%= form_for(@topic) do |f| %>
+  <%= f.invisible_captcha :subtitle, { timestamp: @invisible_captcha_values.timestamp, spinner_value: @invisible_captcha_values.spinner_value } %>
+  <!-- or -->
+  <%= invisible_captcha :subtitle, :topic, { timestamp: @invisible_captcha_values.timestamp, spinner_value: @invisible_captcha_values.spinner_value } %>
+<% end %>
+```
+
+Controller code:
+
+```ruby
+class TopicsController < ApplicationController
+  before_action :invisible_captcha_values, only: %i[new create update]
```

Comment:
Similar to https://github.com/markets/invisible_captcha/pull/85/files#r573288133, would be ideal to do this without the user configuring/adding more lines

Reply:
I agree, made change.